packages/notifi-frontend-client/lib/client/NotifiFrontendClient.ts

```diff
@@ -655,24 +734,20 @@ export class NotifiFrontendClient {
   }
 
   async logIn(
-    signMessageParams: SignMessageParams,
+    signMessageParams: LoginParams,
```

Comment:
Shall we change the name of argument to `loginParams`, so people will not confuse it with the type of `SignMessageParams`